docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -312,52 +255,25 @@ keycloak.init({
-future.
+In the latest versions of some browsers various cookies policies are applied to prevent tracking of the users by third-parties, like SameSite in Chrome or completely blocked third-party cookies. It is expected that those policies will become even more restrictive and adopted by other browsers over time, eventually leading to cookies in third-party contexts to be completely unsupported and blocked by the browsers. The adapter features affected by this might get deprecated in the future.
 
-Javascript adapter relies on third-party cookies for Session Status iframe, _silent_ `check-sso` and partially also for
-regular (non-silent) `check-sso`. Those features have limited functionality or are completely disabled based on how
-the browser is restrictive regarding cookies. The adapter tries to detect this setting and reacts accordingly.
+The adapter relies on third-party cookies for Session Status iframe, _silent_ `check-sso` and partially also for regular (non-silent) `check-sso`. Those features have limited functionality or are completely disabled based on how the browser is restrictive regarding cookies. The adapter tries to detect this setting and reacts accordingly.
 
 ===== Browsers with "SameSite=Lax by Default" Policy
-All features are supported if SSL / TLS connection is configured on the {project_name} side as well as on the application
-side. Affected is for example Chrome starting with
-version 84.
+All features are supported if SSL / TLS connection is configured on the {project_name} side as well as on the application side. Affected is for example Chrome starting with version 84.
 
 ===== Browsers with Blocked Third-Party Cookies
-Session Status iframe is not supported and is automatically disabled if such browser behavior is detected by the JS adapter.
-This means the adapter cannot use session cookie for Single Sign-Out detection and have to rely purely on tokens. This
-implies that when user logs out in another window, the application using JavaScript adapter won't be logged out until it
-tries to refresh the Access Token. Therefore, it is recommended to set Access Token Lifespan to relatively short time, so
-that the logout is detected rather sooner than later. Please see link:{adminguide_link}#_timeouts[Session and Token Timeouts].
+Session Status iframe is not supported and is automatically disabled if such browser behavior is detected by the adapter. This means the adapter cannot use session cookie for Single Sign-Out detection and have to rely purely on tokens. This implies that when user logs out in another window, the application using the adapter won't be logged out until it tries to refresh the Access Token. Therefore, it is recommended to set Access Token Lifespan to relatively short time, so that the logout is detected rather sooner than later. Please see link:{adminguide_link}#_timeouts[Session and Token Timeouts].
```

Comment:
```suggestion
Session Status iframe is not supported and is automatically disabled if such browser behavior is detected by the adapter. This means the adapter cannot use a session cookie for Single Sign-Out detection and must rely purely on tokens. As a result, when a user logs out in another window, the application using the adapter will not be logged out until the application tries to refresh the Access Token. Therefore, consider setting the Access Token Lifespan to a relatively short time, so that the logout is detected as soon as possible. For more details, see the link:{adminguide_link}#_timeouts[Session and Token Timeouts].
```